test/colorado_lottery_test.rb

```diff
@@ -0,0 +1,72 @@
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/contestant"
+require "./lib/game"
+require "./lib/colorado_lottery"
+
+
+class ColoradoLotteryTest < Minitest::Test
+
+  def setup
```

Comment:
Great job using the setup method here to get the program in the right state for testing.